tests/core/test_solve.py

```diff
@@ -861,214 +806,56 @@ def test_conda_downgrade():
-                'channel-2::itsdangerous-0.24-py27_0',
-                'channel-2::markupsafe-1.0-py27_0',
-                'channel-4::pkginfo-1.4.2-py27_1',
-                'channel-2::psutil-5.2.2-py27_0',
-                'channel-2::pycosat-0.6.2-py27_0',
-                'channel-2::pycparser-2.18-py27_0',
-                'channel-2::pyparsing-2.2.0-py27_0',
-                'channel-4::pysocks-1.6.8-py27_0',
-                'channel-4::pyyaml-3.13-py27h14c3975_0',
-                'channel-2::requests-2.14.2-py27_0',
-                'channel-4::ruamel_yaml-0.15.46-py27h14c3975_0',
-                'channel-2::scandir-1.5-py27_0',
-                'channel-2::six-1.10.0-py27_0',
-                'channel-2::cffi-1.10.0-py27_0',
-                'channel-2::packaging-16.8-py27_0',
-                'channel-2::setuptools-36.4.0-py27_1',
-                'channel-2::cryptography-1.8.1-py27_0',
-                'channel-2::jinja2-2.9.6-py27_0',
-                'channel-2::pyopenssl-17.0.0-py27_0',
-                'channel-2::conda-4.3.30-py27h6ae6dc7_0',
-                'channel-4::urllib3-1.23-py27_0',
-                'channel-4::conda-build-3.12.1-py27_0',
+                'channel-2::xz-5.2.3-0',
+                'channel-2::yaml-0.1.6-0',
+                'channel-2::python-3.6.2-0',
+                'channel-2::asn1crypto-0.22.0-py36_0',
+                'channel-4::beautifulsoup4-4.6.3-py36_0',
+                'channel-2::certifi-2016.2.28-py36_0',
+                'channel-4::chardet-3.0.4-py36_1',
+                'channel-4::filelock-3.0.4-py36_0',
+                'channel-4::glob2-0.6-py36_0',
+                'channel-2::idna-2.6-py36_0',
+                'channel-2::itsdangerous-0.24-py36_0',
+                'channel-2::markupsafe-1.0-py36_0',
+                'channel-4::pkginfo-1.4.2-py36_1',
+                'channel-2::psutil-5.2.2-py36_0',
+                'channel-2::pycosat-0.6.2-py36_0',
+                'channel-2::pycparser-2.18-py36_0',
+                'channel-2::pyparsing-2.2.0-py36_0',
+                'channel-2::pyyaml-3.12-py36_0',
+                'channel-2::requests-2.14.2-py36_0',
+                'channel-2::ruamel_yaml-0.11.14-py36_1',
+                'channel-2::six-1.10.0-py36_0',
+                'channel-2::cffi-1.10.0-py36_0',
+                'channel-2::packaging-16.8-py36_0',
+                'channel-2::setuptools-36.4.0-py36_1',
+                'channel-2::cryptography-1.8.1-py36_0',
+                'channel-2::jinja2-2.9.6-py36_0',
+                'channel-2::pyopenssl-17.0.0-py36_0',
+                'channel-2::conda-4.3.30-py36h5d9f9f4_0',
+                'channel-4::conda-build-3.12.1-py36_0'
             )
             assert convert_to_dist_str(unlink_precs) == unlink_order
             assert convert_to_dist_str(link_precs) == link_order
     finally:
         sys.prefix = saved_sys_prefix
 
 
-def test_install_uninstall_features_1():
```

Comment:
why do test_install_uninstall_features_1 and test_install_uninstall_features_2 get removed here? I thought they should pretty much be unaffected by the changes in the PR?

Reply:
these tests are unfortunately totally irrelevant.  None of our packages have any features anymore.  We have hotfixed them all away.  These tests broke because of the new, different pruning behavior.

We need to write new tests if we're going to maintain the features functionality.  I'm inclined to rip it out, but we may need to have that be in 5.0.